lnst/RecipeCommon/Perf/Measurements/Results/AggregatedTcRunMeasurementResults.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+from statistics import mean
+from typing import Union
+
+from lnst.Devices import Device
+from lnst.RecipeCommon.Perf.Measurements.MeasurementError import MeasurementError
+from lnst.RecipeCommon.Perf.Measurements.Results.TcRunMeasurementResults import TcRunMeasurementResults
+from lnst.RecipeCommon.Perf.Results import SequentialPerfResult
+
+
+class AggregatedTcRunMeasurementResults(TcRunMeasurementResults):
+
+    def __init__(
+            self,
+            measurement: "TcRunMeasurement",
+            device: Device,
+            warmup_rules=0,
+    ):
+        super().__init__(measurement, device, warmup_rules=warmup_rules)
+        self._individual_results: list[TcRunMeasurementResults] = []
+
+    @property
+    def description(self):
+        return f"{self.device.host.hostid}.{self.device.name} multi tc run" \
+               f" num_instances={self.num_instances}"\
+               f" mean time_taken={self.time_taken}s" \
+               f" num_rules={self.num_rules}"
+
+    @property
+    def run_interval(self) -> SequentialPerfResult:
+        return SequentialPerfResult([i.run_interval for i in self.individual_results])
+
+    @property
+    def time_taken(self) -> float:
```

Comment:
time taken sounds like the cumulative time taken but this returns a mean. Are we sure that returning the mean here is the correct thing?

If so, would it make sense to maybe rename this to `mean_time_taken` to make it a bit more explicit?

Reply:
Its named that way because its the overriding the `time_taken` property from the base TcRunMeasurementResults.

Reply:
makes sense.